### DISCUSSION TITLE
Is there a chinese model or relevant plan?

### DISCUSSION THREAD

**Discussion by uloveqian2021** [snakers4/silero-vad#101] [Ideas]
## Is there a chinese model or relevant plan?

We have a [wiki](https://github.com/snakers4/silero-models/wiki) available for our users. Please make sure you have checked it out first.

**Comment by snakers4** [comment]
The vad should generalize to all languages reasonably well, though it is trained on 4 or 5.
We are planning a large quality update shortly, that would mostly make parameter tuning much easier.
There are plans to add other 90 languages to VAD training data, but we are still looking for financing for this feature.

**Reply by snakers4** [reply]
See other issues and discussions where people test it on Chinese data

**Reply by uloveqian2021** [reply]
> See other issues and discussions where people test it on Chinese data

ok, thanks!
